src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -544,15 +518,30 @@ private static bool HaveSameReturnTypes(Symbol member1, TypeMap typeMap1, Symbol
             return returnType1.Equals(returnType2, typeComparison);
         }
 
-        private static TypeMap GetTypeMap(Symbol member)
+        private static TypeMap GetTypeMap(Symbol member, Symbol copyIsReferenceTypeFrom)
```

Comment:
>, Symbol copyIsReferenceTypeFrom) [](start = 55, length = 33)

I do not believe the refactoring results in an equivalent behavior. Please restore the original behavior.